### DISCUSSION TITLE
Failed to get the authorisation code

### DISCUSSION THREAD

**Discussion by LeSnow-Ye** [spencerwooo/onedrive-vercel-index#319] [Q&A] [answered]
In [Step 2 - Get authorisation code](https://docs.spencerwoo.com/getting-started#0ad6f8eae120453c81a3d6aa6b5ce228),
my redirected URL appears to be something like this:
`https://localhost/?code=0.AXAAZ8y8rpOul0CrqlZMnJxiPwJcxtIZ5LJKjvET8Cay-_JwAOI.AQABAAIAAAD--DLA3VO7QrddgJg7WevrNa87kYqt-c0KgYjv5UHLOsgJrb35rtEI1y9ohnUDuEk1It7F07BfQYgg9LeQ5BW8g4SAjMjJzxMVwyyyyeNsIWTysrhuhJ2MxEcpLt8xgA7sOUUkapZHThNNjkQxyVkScrVtb36UpEs1yEYEv349xz71Y-d13YeDEJQYKgEqA_8vxmt_39AduufZvGO9NOPQV30rKMpBLwtmiJezAzekJX_P0y0P2H2HRAcNPMNtmpXECDLQuhQFVMsxlkDBAAba8mrLAHVVhu19k71zw0jYWSyX0cqe4u-zd6zFdPfh4yWkQY1nMRM_5i6k3EvxYp42DA3OG5NFv-pTLItov2uucicFtC4kyOQVN4M5e9tpoYuD7XCOSJmvmjDt_9n4YnPFnmgmH4wKqOilRl3Gs0HhCkkxcpuB_PBo1CVdgIA0vZaeZq6wBgKVKj16CMKQDHHoflvv0vODTO-s56jqmdPgbzOihcS8vqY61DfYNzZgiF8EiVMLDMpFJVyGorzgVvTqM9dKyvaoZNOKlzS5ZA0hSMSpVgF_JMr213ePdJbXNmbUn46YbI8gBptXwQPJFem8Fp6JF5zIWPEiiLoeoX3RQCqdD27qk_bhJlOa1p2oNZKI8w0atAZ6qywPnC8DHK4yIAA&session_state=3429b585-43b1-4096-a0b0-67a4b812a0c8#`

And it can not be extracted. What should I do?
I used my E5 account and tried using my own client id and secret

**Comment by spencerwooo** [comment, marked as answer]
`https` -> `http`

(I suppose you are using HTTPS everywhere as well? #310 )
